src/navigation/components/AddButton.tsx

```
@@ -108,13 +119,18 @@ type Props = {
     closeModal: () => void;
 }
 
+const WorkingTouchableOpacity = Platform.OS === "ios"
+  ? TouchableOpacity
+  : Pressable
+
 class AddButtonOptions extends React.PureComponent<Props> {
     constructor(props: Props) {
         super(props);
     }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)